README.md

```diff
@@ -1,14 +1,27 @@
 
 
-# Ansible Network Collection for network and IP utilities that are not specific to any platform or OS, or that interact with any platform os. 
+# Ansible Network Meta Collection.
 [![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/builds?project=ansible-collections%2Fansible.network) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/ansible.network)](https://codecov.io/gh/ansible-collections/ansible.network)-->
 
-The Ansible ``ansible.network`` collection includes common content network and IP utilities that are not specific to any platform or OS.
+The Ansible ``ansible.network`` collection is a meta collection that install all the following network supported content collections.
+ ```
+- ansible.netcommon
+- ansible.utils
+- arista.eos
+- cisco.ios
+- cisco.iosxr
+- cisco.nxos
+- frr.frr
+- junipernetworks.junos
+- openvswitch.openvswitch
+- vyos.vyos
+ ```
+
 
 <!--start requires_ansible-->
 ## Ansible version compatibility
 
-This collection has been tested against following Ansible versions: **>=2.9.10,<2.11**.
+This collection has been tested against following Ansible versions: *****.
```

Comment:
This still has to be at least `>=2.9.10`, right?
```suggestion
This collection has been tested against following Ansible versions: **>=2.9.10**.
```

Reply:
corrected!